kasa/device.py

```diff
@@ -435,7 +435,11 @@ def has_emeter(self) -> bool:
     @property
     @abstractmethod
     def on_since(self) -> datetime | None:
-        """Return the time that the device was turned on or None if turned off."""
+        """Return the time that the device was turned on or None if turned off.
+
+        Could be inprecise by up to 5 seconds due to device jitter between
+        the device reporting time and on_time.
```

Comment:
It's probably better to describe what exactly is happening and why, maybe something like "This returns a cached value if the device reported value difference is under five seconds to avoid device-caused jitter."?

Reply:
That's done